### PR TITLE
Add i2c-bus definitions

### PR DESCRIPTION
**Pull request by 101100**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

File: i2c-bus/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,400 @@
+// Type definitions for i2c-bus v1.0.0
+// Project: https://github.com/fivdi/i2c-bus
+// Definitions by: Jason Heard <https://github.com/101100>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module "i2c-bus" {
```
Comment (vvakame) [inline i2c-bus/index.d.ts]: remove this block.
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
We recommend `ProperModule` kind.

Comment (vvakame) [inline i2c-bus/index.d.ts]: yep.

**Comment by vvakame** [comment]
LGTM. waiting for complete the Travis CI process.

**Comment by 101100** [comment]
@vvakame The latest version has now passed.

**Comment by 101100** [comment]
@vvakame Thanks for your help.

**Pull request merged**